cortex_xdr_client/api/alerts_api.py

```diff
@@ -167,6 +167,98 @@ def get_alerts_v2(
         )
         return GetAlertsResponseV2.model_validate(response.json())
 
+    def get_siem_alerts(
+        self,
+        alert_id_list: list[str] | None = None,
+        creation_time: int | None = None,
+        limit: int = 50,
+        page: int = 0,
+    ) -> GetAlertsResponse | None:
+        """
+        Get a list of SIEM alerts using page-based pagination.
+        This method uses the /alerts/get_alerts endpoint for Cortex XSIAM.
+
+        :param alert_id_list: List of alert IDs (as strings)
+        :param creation_time: Timestamp of the Creation time in milliseconds
+        :param limit: Number of alerts per page (max 100)
+        :param page: Page number (0-indexed)
+        :return: Returns a GetAlertsResponse object if successful.
+        """
+        filters = []
+
+        if alert_id_list is not None:
+            filters.append(request_filter("alert_id_list", "in", alert_id_list))
+
+        if creation_time is not None:
```

Comment:
regarding sort. did you verify it is coming in the right order? oldest first. I remember there is a sort option here as well. should be set specifically, never trust the vendor order unless there is no option to set it